app/actions/ClientActions.js

```diff
@@ -100,6 +104,9 @@ export const getStartupWalletInfo = () => (dispatch) => {
         await dispatch(findImmatureTransactions());
         await dispatch(getAccountsAttempt(true));
         await dispatch(getStartupStats());
+        if (dcrdataEnabled) {
+          await dispatch(getTreasuryBalance());
```

Comment:
You can drop the `await` from here, given that the treasury balance data is not critical to the wallet operation and if it fails (eg: due to dcrdata being offline or unreachable or if there's a change in the api that breaks compatibility) we should still be able to continue with the wallet startup.

Reply:
Done